src/components/Data/Documents/Common/CreateOrUpdate.vue

```diff
@@ -196,7 +197,13 @@ export default {
   methods: {
     onJsonChange(val) {
       this.rawDocument = val
-      this.$emit('document-change', JSON.parse(val))
+      let parsed = {}
+      try {
+        parsed = JSON.parse(val)
+        this.$emit('document-change', parsed)
+      } catch (error) {
+        // invalid json
```

Comment:
If we do not want to log or do anything with this error, then add an explicit comment.
```suggestion
        // Fail silently
```